Sources/App/Migrations/CreateRepository.swift

```diff
@@ -25,3 +25,21 @@ struct CreateRepository: Migration {
         return database.schema("repositories").delete()
     }
 }
+
+
+// FIXME: squash migrations before launch
+struct AddNameOwner: Migration {
+    func prepare(on database: Database) -> EventLoopFuture<Void> {
+        return database.schema("repositories")
+            .field("name", .string)
+            .field("owner", .string)
+            .update()
+    }
+
+    func revert(on database: Database) -> EventLoopFuture<Void> {
+        return database.schema("repositories")
+            .deleteField("name")
+            .deleteField("owner")
+            .update()
+    }
+}
```

Comment:
I've added the new fields as a migration for now. It's probably best to squash all these into a single schema and reprocess everything but maybe for now it's a good exercise to deploy with a schema migration.

Reply:
Either way is fine by me until we get to a public launch. Agree it's not a bad idea to test the migration process though.